### PR TITLE
Feature/cleanup lexer

### PR DESCRIPTION
**Pull request by alexrefshauge**
general cleanup concerning compilation

NOTE: might remove lexer.NewFromFile

### CODE REVIEW

**Review by GustavHus — APPROVED** [review]
LGTM

File: pkg/cmd/root.go [hunk 1]
```diff
@@ -71,7 +68,7 @@ func Execute() {
 func init() {
 	rootCmd.Flags().BoolVarP(&versionBool, "version", "v", false, "shows current version")
 
-	rootCmd.Flags().StringVar(&compileString, "hammer-time", "", "Compile an ygl file to wasm")
+	rootCmd.Flags().StringVar(&compileString, "hammer-time", "", "CompileOld an ygl file to wasm")
```
Comment (TobiasABay) [inline pkg/cmd/root.go:71]: Godt navn, jeg er fan

**Review by TobiasABay — APPROVED** [review]

**Review by TobiasABay — APPROVED** [review]

File: pkg/synthesis/generator.go [hunk 2]
```diff
@@ -17,9 +17,9 @@ func (g *Generator) write(code string, args ...interface{}) {
 }
 
 // GenWrapper starts the generation process from the root Program node.
-func GenWrapper(p *ast.Program) bytes.Buffer {
-	var b bytes.Buffer
-	generator := &Generator{outBuffer: &b, connections: &p.Connections}
+func GenWrapper(p *ast.Program) *bytes.Buffer {
+	b := new(bytes.Buffer)
```
Comment (TobiasABay) [inline pkg/synthesis/generator.go:21]: New

**Review by TobiasABay — APPROVED** [review]

**Review by MathiasFM — APPROVED** [review]

**Review by Androkless — APPROVED** [review]

File: pkg/synthesis/generator.go (hunk 2 above)
Comment (alexrefshauge) [reply pkg/synthesis/generator.go:21]: Ok

**Pull request merged**